src/lib/utils.ts

```diff
@@ -110,3 +117,51 @@ export const shuffleArray = (arrayToShuffle: unknown[]) => {
 
   return array
 }
+
+// PEOPLE CHAIN RELATED
+
+export type AccountInfoIF = {
+  address: string | number | undefined
+  display?: string | number | undefined
+  legal?: string | number | undefined
+  matrix?: string | number | undefined
+  email?: string | number | undefined
+  twitter?: string | number | undefined
+  web?: string | number | undefined
+}
+
+const dataToString = (value: number | string | Binary | undefined) =>
+  typeof value === 'object' ? value.asText() : (value ?? '')
+
+export const mapRawIdentity = (
+  rawIdentity?: DotPeopleQueries['Identity']['IdentityOf']['Value'],
+) => {
+  if (!rawIdentity) return rawIdentity
+  const {
+    info: { display, email, legal, matrix, twitter, web },
+  } = rawIdentity[0]
```

Comment:
Just a suggestion to make it easier to read.
```suggestion
  const { display, email, legal, matrix, twitter, web } = rawIdentity[0].info
```